packages/formstr-app/src/index.css

```diff
@@ -68,3 +68,7 @@ code {
   max-height: 250px;
   overflow-y: scroll;
 }
+
+.ant-dropdown-menu-item {
```

Comment:
why do we need to override this globally? can it also not be a part of the styled div? any advantages to overriding globally?